### PR TITLE
[WIP] Foundations for Multiple Clusters within Common Inventory Files + Centralized Kafka Clusters + Central Control Center Metrics

### PR DESCRIPTION
**Pull request by nerdynick**
# Description

Provides the basics for multiple inter-connected platform/cluster installations with single set of inventory files. 
Provides the logic for a separate/centralized Control Center Metrics cluster. 

Helps to address remaining features outlined in PR #197 that where not addressed in the 6.x updates. 

This PR requires PR #563 prior to approval

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Personal Lab of 2 AWS EC2 based clusters

**Test Configuration**:

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] Any variable changes have been validated to be backwards compatible

### CODE REVIEW

**Comment by nerdynick** [comment]
Be aware, this change looks more significant then it actually is due to the dependence on PR #563

File: roles/confluent.variables/defaults/main.yml [hunk 1]
```diff
@@ -35,11 +35,11 @@ jmxexporter_enabled: false
 jmxexporter_jar_path: /opt/prometheus/jmx_prometheus_javaagent.jar
 
 ### Boolean to have cp-ansible configure components with FIPS security settings
-fips_enabled: false
+fips_enabled: "{{_fips_enabled}}"
 
 fips_jar_path: /usr/share/java/kafka/bc-fips-*.jar
 fips_provider_class: org.bouncycastle.jcajce.provider.BouncyCastleFipsProvider
-pkcs12_enabled: "{{ true if fips_enabled|bool else false }}"
+pkcs12_enabled: "{{ _pkcs12_enabled }}"
```
Comment (domenicbove) [inline roles/confluent.variables/defaults/main.yml:42]: i think i meant to delete these in the fips rework... at the minimum i do not think you need to pay any attention to pkcs12_enabled

File: roles/confluent.variables/defaults/main.yml [hunk 2]
```diff
@@ -35,11 +35,11 @@ jmxexporter_enabled: false
 jmxexporter_jar_path: /opt/prometheus/jmx_prometheus_javaagent.jar
 
 ### Boolean to have cp-ansible configure components with FIPS security settings
```
Comment (domenicbove) [inline roles/confluent.variables/defaults/main.yml:37]: I think i understand whats going on here and i like it, unfortunately the documentation python script i wrote actually prints out the default... so now no one will know the default is false.... the simple solution is just to put in the annotation. this defaults to false

File: roles/confluent.variables/defaults/main.yml (hunk 1 above)
Comment (nerdynick) [reply roles/confluent.variables/defaults/main.yml:42]: It's removed in 6.2.x but not in 6.1.0-post where I'm basing the work from

File: roles/confluent.variables/defaults/main.yml (hunk 2 above)
Comment (nerdynick) [reply roles/confluent.variables/defaults/main.yml:37]: I think I'm actually going to revert this change and go a different route with this. IE force folks to be explicit. All global vars will be treated as fully global. If you want something to be different then whats defined global, you then do the change explicitly to your cluster/resources in question via group/hostvars.

**Comment by nsharma-git** [comment]
@nerdynick Is this PR still valid. If that's the case, can you please resolve the conflict and update it.

**Comment by nsharma-git** [comment]
@nerdynick A friendly reminder

**Comment by nsharma-git** [comment]
Closing this PR as it has conflicts and the owner has not responded since a while. We will raise another PR after resolving the conflicts.

**Pull request closed without merging**